core/src/com/unciv/ui/civilopedia/CivilopediaText.kt

```diff
@@ -48,7 +49,7 @@ class FormattedLine (
     val extraImage: String = "",
     /** Width of the [extraImage], height is calculated preserving aspect ratio. Defaults to available width. */
     val imageSize: Float = Float.NaN,
-    /** Text size, defaults to 18. Use [size] or [header] but not both. */
+    /** Text size, defaults to [Constants.defaultFontSize]. Use [size] or [header] but not both. */
     val size: Int = Int.MIN_VALUE,
```

Comment:
Is this changed in another piece of code, or does this default to `Int.MIN_VALUE` instead of `[Constants.defaultFontSize]`?

Reply:
Hm good point. This SHOULD be set for everyone, but why not avoid problems we didn't think of prematurely?

Reply:
I didn't investigate too closely, but the stuff I saw in my peripheral vision as I did the one-to-one replacements led me to believe that Int.MIN_VALUE as a kind of `null`-like value that gets replaced with the default size.

Reply:
Precisely. Used as placeholder for "not set". Same with Float.NaN just above.